converters/toZigbee.js

```diff
@@ -4828,6 +4828,20 @@ const converters = {
             await entity.read('closuresDoorLock', [0x4005], {manufacturerCode: 4919});
         },
     },
+    schneider_pilot_mode: {
+        key: ['schneider_pilot_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            const lookup = {'relay': 1, 'pilot': 3};
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(lookup));
+            const mode = lookup[value];
+            await entity.write('schneiderSpecificPilotMode', {'pilotMode': mode}, {manufacturerCode: 0x105e});
+            return {state: {schneider_pilot_mode: mode}};
+        },
+        convertGet: async (entity, key, meta) => {
+            await entity.read('schneiderSpecificPilotMode', ['pilotMode'], {manufacturerCode: 0x105e});
```

Comment:
This get doesn't add any value since there is no corresponding fromZigbee converter. Please add one. (you can test it by doing `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"schneider_pilot_mode":""}`